libfreerdp/core/freerdp.c

```diff
@@ -703,9 +703,10 @@ BOOL freerdp_context_new(freerdp* instance)
 		goto fail;
 	}
 
-	update_register_client_callbacks(rdp->update);
 	instance->context->abortEvent = CreateEvent(NULL, TRUE, FALSE, NULL);
 
+	transport_register_default_io_callbacks(rdp->update);
```

Comment:
Is this still required since the `update_new` function already registers this?

Reply:
no, just missed this.